shell/platform/darwin/macos/framework/Source/FlutterOpenGLRenderer.mm

```diff
@@ -0,0 +1,187 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterOpenGLRenderer.h"
+#include "flutter/shell/platform/darwin/macos/framework/Source/FlutterView.h"
+#include "flutter/shell/platform/embedder/embedder.h"
+
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterEngine_Internal.h"
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterExternalTextureGL.h"
+
+#pragma mark - Static methods for openGL callbacks that require the engine.
+
+static bool OnMakeCurrent(FlutterEngine* engine) {
+  return [engine.openGLRenderer makeCurrent];
+}
+
+static bool OnClearCurrent(FlutterEngine* engine) {
+  return [engine.openGLRenderer clearCurrent];
+}
+
+static bool OnPresent(FlutterEngine* engine) {
+  return [engine.openGLRenderer present];
+}
+
+static uint32_t OnFBO(FlutterEngine* engine, const FlutterFrameInfo* info) {
+  return [engine.openGLRenderer fboForFrameInfo:info];
+}
+
+static bool OnMakeResourceCurrent(FlutterEngine* engine) {
+  return [engine.openGLRenderer makeResourceCurrent];
+}
+
+static bool OnAcquireExternalTexture(FlutterEngine* engine,
+                                     int64_t textureIdentifier,
+                                     size_t width,
+                                     size_t height,
+                                     FlutterOpenGLTexture* openGlTexture) {
+  return [engine.openGLRenderer populateTextureWithIdentifier:textureIdentifier
+                                                openGLTexture:openGlTexture];
+}
+
+#pragma mark - FlutterOpenGLRenderer implementation.
+
+@implementation FlutterOpenGLRenderer {
+  // The embedding-API-level engine object.
+  FLUTTER_API_SYMBOL(FlutterEngine) _engine;
+
+  FlutterView* _flutterView;
+
+  // The context that is owned by the currently displayed FlutterView. This is stashed
```

Comment:
Is this comment still accurate? This class is creating the context.

Reply:
This was valid during an earlier iteration, fixed it now.